flash/core/model.py

```diff
@@ -60,8 +60,8 @@ class Task(LightningModule):
         optimizer: Optimizer to use for training, defaults to `torch.optim.Adam`.
         metrics: Metrics to compute for training and evaluation.
         learning_rate: Learning rate to use for training, defaults to `5e-5`.
-        default_preprocess: :class:`.Preprocess` to use as the default for this task.
-        default_postprocess: :class:`.Postprocess` to use as the default for this task.
+        preprocess: :class:`.Preprocess` to use as the default for this task.
```

Comment:
```suggestion
        preprocess: :class:`Preprocess` to use as the default for this task.
```

Reply:
I think this would stop sphinx from generating the links properly?